rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -0,0 +1,84 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_maintain
+%global confdir foreman_maintain
+
+%{!?_root_bindir:%global _root_bindir %{_bindir}}
+
+Summary: The Foreman/Satellite maintenance tool
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.2
+Release: 1%{?dist}
+Group: Development/Languages
+License: GPLv3
+URL: http://github.com/theforeman/foreman_maintain
```

Comment:
This would be the final URL when @iNecas moves the project under Foreman project org.